common/src/main/java/com/microsoft/identity/common/adal/internal/AuthenticationConstants.java

```diff
@@ -1059,6 +1059,16 @@ private AuthenticationConstants() {
          */
         public static final String CLIENT_APP_PACKAGE_NAME = "client_app_package_name";
 
+        /**
+         * String of account environment key.
+         */
+        public static final String ENVIRONMENT_KEY = "environment.key";
+
+        /**
+         * String to return account list from broker.
+         */
+        public static final String BROKER_ACCOUNTS = "broker.accounts";
```

Comment:
Sorry should have been explicit, iOS and we agreed to name broker constants snake cased. I mean values
` public static final String ENVIRONMENT= "environment"` 
`  public static final String BROKER_ACCOUNTS = "broker_accounts"`